src/inventory/gilded_rose.js

```diff
@@ -22,49 +22,77 @@ const items = [
 updateQuality(items);
 */
 export function updateQuality(items) {
+    //combine these into quality changer--takes in a + or - number and you can get BOTH increase or decrease in quality
+    qualityUpdater = (item, change_Amount) =>{
+       if (item.quality < 50 && item.quality >0){
+      item.quality = item.quality + change_Amount
+    }
```

Comment:
Don't forget to properly format your code! Like we chatted about yesterday, this makes it difficult to define the boundaries of a function!

Reply:
I am still struggling with this. I would like to find a tutorial on formatting and practice getting better at this so it is second nature. I'm putting that on my to do list and lattice goals. Thank you!